magellan-library/src/main/java/com/wealthfront/magellan/transitions/DefaultTransition.java

```diff
@@ -15,6 +19,25 @@
 
 public class DefaultTransition implements Transition {
 
+  private TimeInterpolator customGoInterpolator = new AccelerateDecelerateInterpolator();
+  private TimeInterpolator customShowInterpolator = new DecelerateInterpolator();
+  private TimeInterpolator customHideInterpolator = new AccelerateInterpolator();
+
+  public DefaultTransition() { }
+
+  public DefaultTransition(TimeInterpolator customInterpolator) {
+    this.customGoInterpolator = customInterpolator;
+    this.customShowInterpolator = customInterpolator;
+    this.customHideInterpolator = customInterpolator;
+  }
+
+  public DefaultTransition(TimeInterpolator customGoInterpolator, TimeInterpolator customShowInterpolator,
```

Comment:
Please use nested constructor calls instead of code duplication

(previous constructor should call this one using this()